examples/integration_test.py

```diff
@@ -1152,18 +1258,20 @@ def print_usage(exitcode, reason=None):
     # Parse options
     while len(sys.argv) > 1 and sys.argv[1].startswith('--'):
         opt = sys.argv.pop(1)[2:]
+
+        if opt == 'conf':
```

Comment:
this is fine, but it should default to "tests/testconf.json" if no config file was specified.

Reply:
tests/testconf-example.json